### PR TITLE
Added script to plot the density-pressure plane

### PR DESCRIPTION
**Pull request by MatthieuSchaller**

### CODE REVIEW

File: plotting/density_pressure.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+
+def setup_axes():
+    """
+    Creates the figure and axis object.
+    """
+    fig, ax = plt.subplots(1)
+
+    ax.set_xlabel("Density [$n_H$ cm$^{-3}$]")
+    ax.set_ylabel("Pressure [K cm$^{-3}$]")
```
Comment (JBorrow) [inline plotting/density_pressure.py]: This should be "Pressure $P / k_B$ [K cm$^{-3}$]"

File: plotting/density_pressure.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    """
+    Grabs the data (P in Kelvin / cm^3 and density in mh / cm^3).
+    """
+
+    data = load(filename)
+
+    density_factor = float(data.gas.densities.cosmo_factor.a_factor)
+    pressure_factor = float(data.gas.pressures.cosmo_factor.a_factor)
+
+    number_density = (data.gas.densities * (density_factor / mh)).to(cm ** -3)
```
Comment (JBorrow) [inline plotting/density_pressure.py]: See `density_internal_energy` for a better version of the cosmology transforms

File: plotting/density_pressure.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+from swiftsimio import load
+
+from unyt import mh, cm, kb, K
+from matplotlib.colors import LogNorm
+from matplotlib.animation import FuncAnimation
+
+# Constants; these could be put in the parameter file but are rarely changed.
+density_bounds = [10 ** (-9.5), 1e6]  # in nh/cm^3
+pressure_bounds = [10 ** (-4), 10 ** (10)]  # in K/cm^2
```
Comment (JBorrow) [inline plotting/density_pressure.py]: Units are inconsistent - are they K / cm^2 or K / cm^3?

File: plotting/density_pressure.py (hunk 2 above)
Comment (MatthieuSchaller) [reply plotting/density_pressure.py]: I had based it off the one in the rho-T script. But indeed, this one is better.

**Comment by JBorrow** [comment]
You can do the same thing for the pressure cosmology :)

**Pull request merged**